assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -31,18 +31,29 @@ import {
 	PropertySelect,
 	UseSnippetSwitch,
 } from '../../../../analytics-4/components/common';
+import ErrorNotice from '../../../../../components/ErrorNotice';
+import SpinnerButton from '../../../../../components/SpinnerButton';
+import {
+	MODULES_ANALYTICS_4,
+	PROPERTY_CREATE,
+} from '../../../datastore/constants';
 import useExistingTagEffect from '../../../../analytics-4/hooks/useExistingTagEffect';
 import { MODULES_ANALYTICS } from '../../../../analytics/datastore/constants';
-import { MODULES_ANALYTICS_4 } from '../../../datastore/constants';
 import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
 import { ACTIVATION_ACKNOWLEDGEMENT_TOOLTIP_STATE_KEY } from '../../../constants';
 import { useTooltipState } from '../../../../../components/AdminMenuTooltip/useTooltipState';
 import { useShowTooltip } from '../../../../../components/AdminMenuTooltip/useShowTooltip';
 import { AdminMenuTooltip } from '../../../../../components/AdminMenuTooltip/AdminMenuTooltip';
 import { getBannerDismissalExpiryTime } from '../../../utils/banner-dismissal-expiry';
-const { useSelect } = Data;
+const { useDispatch, useSelect } = Data;
+
+export default function SetupBanner( { onSubmitSuccess } ) {
+	const [ errorNotice, setErrorNotice ] = useState( null );
+
+	const ga4MeasurementID = useSelect( ( select ) =>
```

Comment:
As discussed I am not sure this is needed now...

Reply:
I've removed this, thank you!